Dockerfile

```diff
@@ -97,7 +97,7 @@ COPY scripts/obodash /tools
 RUN chmod +x /tools/obodash && \
```

Comment:
> Fuck it, we are _not_ going to create a virtual environment just for the
OBO Dashboard. If the OBO Dashboard dependencies end up breaking the
rest of our Python packages, we might as well just create a separate
Docker image just for the dashboard.

Your commit messages are great, but it's important to avoid using offensive language in them. Thank you!

Reply:
Sorry but I won’t take lessons in how to write commit messages from folks who can’t be bothered to write decent commit messages themselves.

If people complain about an occasional profanity in one of my commit messages, at least I will know that there _are_ some people who read them.

Reply:
Correct me if I am wrong but I dont think this was meant as a lesson, more like a statement of affectedness. 

I also prefer to avoid strong language in the spirit of conduct (as the Covenant Code of Conduct suggests), I had endless discussions of misconduct in some other projects which just distract from our common goals.

Reply:
> Correct me if I am wrong

My pleasure.

> I dont think this was meant as a lesson, more like a statement of affectedness.

A “statement of affectedness” would have been something like, “I felt offended by the presence of that F-word, I don’t like reading such things when I am reviewing a PR. I’d appreciate if you abstained to do that“ or similar.

“It’s important to avoid using offensive language in [commit messages]” is absolutely a lesson in how one should write a commit message – one that I find hard to accept coming from someone who I can’t remember having ever seen write a single decent commit message that actually explains what they are trying to do.

> as the Covenant Code of Conduct suggests

The ODK does _not_ have a code of conduct. Maybe it should (though I’d much rather have a “code on how to write efficient commit messages so that future contributors don’t have to guess what the f... the developers had in mind when they committed that change”, but it’s OK, we can have different priorities), but currently it does not.

Even if it had, according to the most popular code of conducts in use out there, the correct response to the occasional use of inappropriate language is a “*private*, written warning from community leaders, providing clarity around the nature of the violation and an explanation of why the behavior was inappropriate” – not a public rebuke.

> I had endless discussions of misconduct in some other projects which just distract from our common goals

Fine. I hereby publicly apologise for having expressed my state of mind, as to why I believe installing the OBO Dashboard in a virtual environment just to circumvent `pip`’s default behaviour would be a unnecessary complication, using a 6-letters profanity instead of a more elaborate, professional, and polite phrasing. I did that for the sake of efficiency, but I didn’t consider that indeed people might not want to read a F-word when they are busy reviewing a PR.

I hope this will end the current distraction, and I will temporarily withdraw for all ODK maintenance to be sure you are not distracted again.